src/Compute/Compute.Test/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,33 @@
+Invoke-LiveTestScenario -Name "VM.NewVM Test" -Description "Test create new VM" -ScenarioScript `
```

Comment:
The Name here looks like does not match what actually will do in the script. Can you please modify it to reflect the real purpose of this live test case?